garak/generators/huggingface.py

```diff
@@ -505,18 +554,23 @@ def __init__(
                 self.name, padding_side="left"
             )
 
-        self.do_sample = do_sample
         self.generation_config = transformers.GenerationConfig.from_pretrained(
             self.name
         )
         self.generation_config.eos_token_id = self.model.config.eos_token_id
         self.generation_config.pad_token_id = self.model.config.eos_token_id
 
+    def _clear_client(self):
```

Comment:
Do we also want to clear `self.tokenizer` here?

Reply:
Good catch! Thank you